src/main/java/frc/robot/subsystems/ArmSystem.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.*;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class ArmSystem extends SubsystemBase {
+    private final CANSparkMax motor;
+    private final RelativeEncoder neoEncoder;
+    private final DutyCycleEncoder dutyCycleEncoder;
+
+    public ArmSystem() {
+        motor = new CANSparkMax(RobotMap.ARM_MOTOR_PORT, CANSparkLowLevel.MotorType.kBrushless);
+        neoEncoder = motor.getAlternateEncoder(RobotMap.ARM_ENCODER_CPR);
+        dutyCycleEncoder = new DutyCycleEncoder(RobotMap.ARM_ENCODER_PORT);
```

Comment:
Because we aim to use the PIDF in the SparkMax for future parts, you should:

sync the NEO encoder to the absolute encoder value, so that they show the same values. This will allow us to use the _SparkMAX_ PIDF (which uses the NEO encoder) while preserving the post boot capabilities of the absolute encoder. So write the code and make sure that they show the same values.

Reply:
You want that i do the same thing that i did with duty cycle encoder? or there is something else?

Reply:
done

Reply:
- You can't call `setPosition` with angle information which `getDutyCycleEncoderPosition` returns. You need to supply the data in native encoder data, which are motor rotations.
- You probably should do this after `restoreFactoryDefaults` in case it affects it.

Reply:
done